streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -95,6 +98,28 @@ public void cleanup() {
         }
     }
 
+    @Test
+    public void testOsDefaultSocketBufferSizes() {
+        props.put(CommonClientConfigs.SEND_BUFFER_CONFIG, Selectable.USE_DEFAULT_BUFFER_SIZE);
+        props.put(CommonClientConfigs.RECEIVE_BUFFER_CONFIG, Selectable.USE_DEFAULT_BUFFER_SIZE);
+        final KafkaStreams streams = new KafkaStreams(builder.build(), props);
+        streams.close();
```

Comment:
Just catching up on this PR. Should we wrap the `close()` into a `finally` block?